### ISSUE TITLE
TB Print Preview Dialog doesn't work when Conversations Add-on is enabled

### ISSUE THREAD

**Issue by WalZyl** [thunderbird-conversations/thunderbird-conversations#1804]
* **Thunderbird version:** 91.6.2
* **Thunderbird Conversation version:** 4.0.12
* **Operating System:** Windows 10 Pro x64bit

**What did you do?** A stand alone message (not in thread, not collapsed) is selected. Neither Ctrl-P, nor Hamburger menu Print ..., nor Menu Print ...., nor Print ... after right-clicking on message is starting the print preview dialog. 

No errors and no warnings pop-up in the error console.
After disabling Conversations add-on the print preview dialog is displayed. 

**What did you expect to happen?** Print preview dialog appears

**What actually happened?** Nothing

**Comment by WalZyl** [comment]
More specific additional observations while TB conversation is enabled.

Print Preview Dialog for a selected stand alone message ...
1.  ... does **not** open if the Message Pane (F8) is closed **AND** the message has not been opened in the Message Pane before  
2.  ... does open if the the message is or has been before displayed in the Message Pane (F8) - which can be closed after displaying
3.  ... does **not** open if the message is or has been displayed before in a new tab via Open Message (Ctrl-O) or Open Conversation Message (Ctrl-Shift-O); regardless of the message tab is still open or already closed.

In summary, if the messaged has been or is displayed in the message pane (F8), Print Preview Dialog works fine.

**Comment by Standard8** [comment]
I've not been able to reproduce this. I've even tried restarting Thunderbird with the message pane closed, and it works every time.

I am on Thunderbird 91.7.0 though, so I don't know if that makes a difference?

**Comment by WalZyl** [comment]
I'm on Thunderbird 91.7.0 too. This is still persistent.

Particularly (after looking in more detail at my use case) it occurs, if a message is open in a tab; at least one tab with a message should be open. Then, I change to message list and close Thunderbird with message pane closed. After Re-start: Ctrl-P does not work. If, however, there is no tab with a message open before closing Thunderbird, after re-start Ctrl-P does work.

Can you reproduce this ?

**Comment by Standard8** [comment]
Thank you for those additional steps. I think I've just fixed this in the source - if the message pane is not open, then we'll defer to Thunderbird to handle the printing.

It might be a week or so before a version comes out with a fix.
